src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -36,7 +38,57 @@
 JOBS_MISSING_ERR = "No valid job IDs provided"
 
 CELLS_NOT_PROVIDED_ERR = "cell_id_list not provided"
-DOES_NOT_EXIST = "does_not_exist"
+
+
+class OutputStateErrMsg(Enum):
+    """
+    For errors that go into the STATUS response.
+    Enum mapping from error names to formattable error messages.
+    The first formatting argument of each error message must be the job ID
+    """
+
+    NOT_FOUND = "Cannot find job with ID %s"
+    NOT_UPDATED = "Job with ID %s has not been updated since ts %d"
+    QUERY_EE2_STATES = "A Job.query_ee2_states error occurred for job with ID %s: %s"
+    CANCEL = "An EE2.cancel_job error occurred for job with ID %s: %s"
+
+    def gen_err_msg(self, its: List[Union[str, int, Iterable]]):
+        """
+        Create a generator for filled in enum values
+
+        :param its:     A list of arguments, where each argument
+                        should be either a literal or an iterable of literals
+                        The iterables will then be zipped and used to format the enum values.
+                        The first argument must be the list of job_ids
+        """
+        if not isinstance(its, list):
+            raise TypeError(
+                "Argument its must be of type list"
+            )
+        # check number of arguments matches number of %s or %d to be formatted
+        num_format = self.value.count("%")
+        if num_format != len(its):
+            raise ValueError(
+                f"{self.__class__.__name__}.{self.name} must be formatted with {num_format} argument(s). "
+                f"Received {len(its)} argument(s)"
+            )
+
+        for i, e in enumerate(its):
+            # if element is a literal, convert to iterable of literal
+            if isinstance(e, str) or isinstance(e, int):
+                its[i] = cycle([e])
+
+        def gen():
+            for tup in zip(*its):
+                yield self.value % tup
+
+        return gen()
+
+    def replace_result(self) -> bool:
```

Comment:
Tells whether the particular enum error is meant to replace the dict in the results dict, or if if is meant to be added onto the existing dict